src/Common/Core/Impl/Imaging/IImageService.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.Common.Core.Imaging {
+    /// <summary>
+    /// Provides access to images and icons
+    /// </summary>
+    public interface IImageService {
```

Comment:
This service looks like 100% platform-specific, you can't use it without type cast. Do you need it anywhere in xplat code?